.cirrus.yml

```diff
@@ -36,6 +36,7 @@ env:
     SPECIAL_FEDORA_CACHE_IMAGE_NAME: "xfedora-30-${_BUILT_IMAGE_SUFFIX}"
     UBUNTU_CACHE_IMAGE_NAME: "ubuntu-19-${_BUILT_IMAGE_SUFFIX}"
     PRIOR_UBUNTU_CACHE_IMAGE_NAME: "ubuntu-18-${_BUILT_IMAGE_SUFFIX}"
+    CENTOS_7_NAME: "centos-7-v20200618"
```

Comment:
I'm having a hard time understanding what this accomplishes. Is it a holdover from an earlier iteration of this PR? And if it is, would it make sense to remove the unused UBUNTU_* declarations?

Reply:
I really hate how github handles comments on stale diffs

Reply:
yes, hold over